modules/ROOT/pages/configure-client-management-openid-task.adoc

```diff
@@ -36,18 +37,37 @@ The *Add OIDC client provider* page appears.
-
-   **** The `Authorization:` header is included as part of the request only if the `Authorization Header` under *Advanced settings* is configured.
-   **** The `client_id` request parameter that is passed is the same as the `client_id` passed in the `PUT` request payload.
-   **** The `token_endpoint_auth_method` passed in the payload is always `client_secret_basic`.
-   **** Only the `Client Registration URL` and `Authorization Header` come from what is configured in OIDC *Client Provider*.
-   +
+** *Client Request Timeout (seconds)*: The time elapsed before a client provider request times out. The minimum value is five seconds. The maximum value is 20 seconds.
+* *Token Introspection Client*
+ ** *Client ID*: The client ID for an existing client in your IdP capable of introspection of _all_ tokens from all clients.
+  *** For Okta, this value should be a "Confidential" client.
+  *** For ForgeRock, this value should be a "Confidential" client.
+  *** For Salesforce, this value should be a "Confidential" client.
+ ** *Client Secret*: The client secret that corresponds to the client ID.
+* *OpenID Connect Authorization URLs*
+ ** *Authorize URL*: The URL where the user authenticates and grants OpenID Connect client applications access to the user's identity.
+ ** *Token URL*: The URL that provides the user’s identity, encoded in a secure JSON Web Token.
+. Click *Create*.
+
+In *Advanced Settings* you can also select:
+
+* *Disable server certificate validation*: Disables server certificate validation if your OpenID client management instance presents a self-signed certificate, or one signed by an internal certificate authority.
+* *Enable client deletion in Anypoint Platform*: Enables deletion of clients created with this integration.
+* *Enable client deletion and updates in IdP*: To use this option, you must also select the *Enable client deletion in Anypoint Platform* option. This option enables you to update and delete external clients in the configured IdP through an outbound call made by Anypoint Platform to `{clientRegistrationUrl}/{clientID}`. The `clientRegistrationUrl` is the value you configured in *Client Registration URL*. For example: `+https://identity.example.com/oauth2/connect/register/6779ef20e75817b79602+`. For detailed example, please see below.
+
+Once this is successfully configured, you can apply the OpenID Connect OAuth Token Enforcement policy to your API Gateways through API Manager. Requesting API access through API portals dynamically generates client applications in the configured IDP that acts as a token provider.
+
+[NOTE]
+For Okta, the Okta admin needs to assign the dynamically generated clients to a user or a group of users in order for them to receive access tokens by sending over the Client ID and Client Secret.
```

Comment:
I wouldn't put two notes in a row - is this something that can be moved higher in the article?

Reply:
Or combined somewhere

Reply:
Good question.. I just copied the previous docs and they were together. Let me try moving one.